tests/syscall/getrandom/enc/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+set(EDL_FILE ../getrandom.edl)
+
+add_custom_command(
+  OUTPUT getrandom_t.h getrandom_t.c
+  DEPENDS ${EDL_FILE} edger8r
+  COMMAND
+    edger8r --trusted ${EDL_FILE} --search-path ${PROJECT_SOURCE_DIR}/include
+    ${DEFINE_OE_SGX} --search-path ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Shouldn't this line be inside some sort of SGX-only check?

Reply:
`DEFINE_OE_SGX` is only set for SGX build: https://github.com/openenclave/openenclave/blob/55e5a29e858e8c6e46940c7d833db6c08046ef80/tests/CMakeLists.txt#L28